### DISCUSSION TITLE
Free Copilot for Students but no one can verify email.

### DISCUSSION THREAD

**Discussion by AlexanderCarlisi** [community/community#110913] [GitHub Education]
My school doesn't allow non school emails to mail to our school email. So I cannot verify my school email is mine.

When I tried to verify im a student even after providing my school ID I was denied access because I couldn't verify my school email, but not a single person in my school can, even after talking to our IT department and principals. 

What can we do to access copilot for free as students?

**Comment by SpiritAxolotl** [comment]
I feel you so much on this one. Good luck solving the issue.
